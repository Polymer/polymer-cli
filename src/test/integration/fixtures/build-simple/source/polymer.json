{
  "builds": [
    {
      "name": "unbundled-uncompiled",
      "bundle": false,
      "js": {
        "compile": false,
        "minify": {
          "exclude": [
            "*.html*",
            "bar.js"
          ]
        }
      },
      "css": {
        "minify": false
      },
      "html": {
        "minify": false
      }
    },
    {
      "name": "bundled-compiled",
      "bundle": true,
      "js": {
        "compile": {
          "exclude": [
            "foo.js"
          ]
        },
        "minify": {
          "exclude": [
            "bar.js"
          ]
        }
      },
      "css": {
        "minify": true
      },
      "html": {
        "minify": true
      }
    }
  ]
}
