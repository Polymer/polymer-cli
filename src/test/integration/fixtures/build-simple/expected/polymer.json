{
  "entrypoint": "index.html",
  "fragments": [],
  "sources": [
    "src/**/*",
    "index.html"
  ],
  "extraDependencies": [
    "bower_components/webcomponentsjs/*.js"
  ],
  "builds": [
    {
      "name": "unbundled-uncompiled",
      "bundle": false,
      "js": {
        "compile": false,
        "minify": {
          "exclude": [
            "*.html*",
            "bar.js"
          ]
        }
      },
      "css": {
        "minify": false
      },
      "html": {
        "minify": false
      }
    },
    {
      "name": "bundled-compiled",
      "bundle": true,
      "js": {
        "compile": {
          "exclude": [
            "foo.js"
          ]
        },
        "minify": {
          "exclude": [
            "bar.js"
          ]
        }
      },
      "css": {
        "minify": true
      },
      "html": {
        "minify": true
      }
    }
  ]
}
